src/css/custom.css

```diff
@@ -105,6 +106,7 @@ Use these if you need'em
 
 [data-theme="dark"] {
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
+  --invert-filter: invert(100%);
```

Comment:
Works as intended. Considering add this to global css configuration? This would allow us to use this invert effect anywhere.

Reply:
Got that.
However not all images need inverting, like colored characters and pictures. Thus I think use a class is a bit better than global application, for the former makes us able to *select* concerning images.

Reply:
Alright.